CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ include(FeatureSummary)
 # Needed for https://github.com/robotology/idyntree/issues/1065
 find_package(Eigen3 REQUIRED)
 find_package(iDynTree REQUIRED)
+find_package(yaml-cpp REQUIRED)
+find_package(LibXml2 REQUIRED)
+
+find_path(RAPIDCSV_INCLUDE_DIRS "rapidcsv.h")
```

Comment:
`find_package(rapidcsv)` isn't working?

Reply:
Since it's header only, it seems to be the suggested way to include it.

Reply:
Apparently rapidcsv does not ship any `rapidcsv-config.cmake` file or similar.

Reply:
I chose it for its extreme simplicity, but we can change it if preferred. However, I think it is possible to get the csv data directly from Creo (eg the joint limits), so at some point I think we could drop the csv support.